Doc/library/ensurepip.rst

```diff
@@ -75,14 +75,14 @@ through two additional command line options:
    for the current Python installation (this option is not permitted inside an
    active virtual environment).
 
-By default, the scripts ``pipX`` and ``pipX.Y`` will be installed (where
-X.Y stands for the version of Python used to invoke ``ensurepip``). The
+By default, the scripts ``pipN`` and ``pipN.Y`` will be installed (where
```

Comment:
Maybe `pipN.M`?

Reply:
Why change this at all? `pipX.Y` is the normal convention here, used in the pip documentation.

Reply:
Ok, this change is unrelated anyway and no users complained about it. I agree with @pfmoore